src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingThrows.cs

```diff
@@ -1,40 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Xunit;
-
-namespace System.Globalization.Tests
-{
-    public class IdnMappingThrows
-    {
-        [Fact]
-        public static void GetAsciiThrows()
-        {
-            IdnMapping idnMapping = new IdnMapping();
-
-            Assert.Throws<ArgumentNullException>("unicode", () => idnMapping.GetAscii(null, -5));
-            Assert.Throws<ArgumentNullException>("unicode", () => idnMapping.GetAscii(null, -5, -10));
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => idnMapping.GetAscii("abc", -5, -10));
```

Comment:
What happened to this test that has both an out -of-range index and an out-of-range count?